cmd/opm/alpha/bundle/build.go

```diff
@@ -15,6 +15,8 @@ var (
 	channelDefaultArgs string
 	outputDirArgs      string
 	overwriteArgs      bool
+	synthesizeCSVArgs  bool
```

Comment:
super nit: Why did we decided to use the word `synthesize` instead of just `create`? Feels like `create` is more intuitive whereas `synthesize` requires a second thought, just coz we're used to seeing create everywhere.
context: simple UX experience.

Reply:
Uhh, idk, I've been using `synthesize` everywhere, but yeah, i can see users would know right away what `create` is better.